code/modules/mob/living/simple_animal/abnormality/waw/parasite_tree.dm

```diff
@@ -0,0 +1,382 @@
+
+/mob/living/simple_animal/hostile/abnormality/parasite_tree/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time, canceled)
+	if(work_type != ABNORMALITY_WORK_REPRESSION && user.stat != DEAD)
+		if(user.has_status_effect(THE_LIARS_BLESSING))
+			var/mob/living/carbon/human/witness = locate(/mob/living/carbon/human) in livinginview(1, src) //included mostly for lore since you encourage people to use it.
+			if(witness && !witness.has_status_effect(THE_LIARS_BLESSING))
+				witness.apply_status_effect(THE_LIARS_BLESSING)
+				to_chat(witness, "<span class='notice'>The air around [src] makes you feel at peace.</span>")
+		else if(datum_reference.qliphoth_meter <= 0)
+			user.apply_status_effect(THE_TREE_CURSE)
+		else
+			user.apply_status_effect(THE_LIARS_BLESSING)
+			if(prob(5))
+				to_chat(user, "<span class='notice'>You hear a voice telling you to bring anyone who needs help here.</span>")
+	else if(work_type == ABNORMALITY_WORK_REPRESSION && datum_reference.qliphoth_meter <= 4)
+		if(canceled || pe < datum_reference.neutral_boxes)
+			to_chat(user, "<span class='notice'>You feel refreshed after being near [src].</span>")
+			if(datum_reference.qliphoth_meter >= 1)
+				user.apply_status_effect(THE_LIARS_BLESSING)
+			else
+				user.apply_status_effect(THE_TREE_CURSE)
+			return ..()
+		else if(locate(THE_TREE_CURSE) in blessed)
+			resetQliphoth()
+			for(var/datum/status_effect/parasite_tree_curse/curse in blessed)
+				qdel(curse)
+			to_chat(user, "<span class='nicegreen'>The scarlet red eye closes as you smash apart [src]'s flowers.</span>")
+			return ..()
+		to_chat(user, "<span class='notice'>[src] stands silently as you finish destroying the buds.</span>")
+		resetQliphoth()
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/parasite_tree/ZeroQliphoth(mob/living/carbon/human/user)
+	if(blessed.len > 0)
+		cut_overlays()
+		var/mutable_appearance/colored_overlay = mutable_appearance(icon, "parasitetreeeye", layer + 0.1)
+		add_overlay(colored_overlay)
+		for(var/datum/status_effect/parasite_tree_blessing/P in blessed)
+			P.facadeFalls()
+	else
+		datum_reference.qliphoth_change(1)
+
+/mob/living/simple_animal/hostile/abnormality/parasite_tree/OnQliphothChange(mob/living/carbon/human/user)
+	cut_overlays()
+	var/budlayer = layer + 0.1
+	var/budtype
+	switch(datum_reference.qliphoth_meter)
+		if(1)
+			budtype = "buds4"
+		if(2)
+			budtype = "buds3"
+		if(3)
+			budtype = "buds2"
+		if(4)
+			budtype = "buds1"
+		else
+			return
+	var/mutable_appearance/colored_overlay = mutable_appearance(icon, budtype, budlayer)
+	add_overlay(colored_overlay)
+
+/mob/living/simple_animal/hostile/abnormality/parasite_tree/proc/resetQliphoth() //unsure if i could of just done datum_reference.qliphoth_meter = start_qliphoth but i wanted the OnQliphothChange to proc too.
+	var/restore_variable = start_qliphoth - datum_reference.qliphoth_meter
+	datum_reference.qliphoth_change(restore_variable)
```

Comment:
if this is to make it so that it fully restores the qliphoth counter, then this is redundant and you can just do `datum_reference.qliphoth_change(9)` or whatever other number is equal to or greater than the max qliphoth counter. If that is not the case, I have no issue with this proc.

Reply:
Oh yeah i can do that instead. I didnt know there was a "hard max" qliphoth set by the starting qliphoth.